src/components/Terminal/Terminal.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useRef, useState } from "react";
+
+import cleanUpTerminal from "./cleanup-terminal";
+import setupTerminal from "./setup-terminal";
+
+import "../../../node_modules/xterm/css/xterm.css";
```

Comment:
`sass-loader` supports `~` as an alias for node_modules, does that work with the css-loader?

Reply:
It doesn't appear to work with the css-loader unfortunately :(